news/2013/2013-09-06-august-2013-highlights.md

```diff
@@ -9,22 +9,22 @@ August was a hectic month for osu!, with the largest yet DDoS attack launched ag
 
 ### A new version-based changelog display is out!
 
-It's now much easier to see what features and bugfixes were in each release! After an update, you can click the notification in the lower right corner of the game client, or go to the main changelog and click one of the buttons up top with the number of the version you want to check out. This will bring up a list of every fix pushed out to that public release, which is quite a step up from the normal changelog when you want to know what updates have just gone live!  
-  
+It's now much easier to see what features and bugfixes were in each release! After an update, you can click the notification in the lower right corner of the game client, or go to the main changelog and click one of the buttons up top with the number of the version you want to check out. This will bring up a list of every fix pushed out to that public release, which is quite a step up from the normal changelog when you want to know what updates have just gone live!\
+\
 
 ### Bancho is now better protected from DDoS attacks!
 
-Due to the high frequency of attacks on Bancho in the last month or so, peppy took it upon himself to port Bancho to HTTP (website format), rather than the much harder to defend TCP. What this means for the end-user is that Cloudflare can now defend Bancho against DDoS attacks, which is a much more powerful protection service than what was available before, and stops almost any attack in its tracks. For more information, look forward to a post on [peppy's blog](https://blog.ppy.sh) about this, coming soon!  
-  
+Due to the high frequency of attacks on Bancho in the last month or so, peppy took it upon himself to port Bancho to HTTP (website format), rather than the much harder to defend TCP. What this means for the end-user is that Cloudflare can now defend Bancho against DDoS attacks, which is a much more powerful protection service than what was available before, and stops almost any attack in its tracks. For more information, look forward to a post on [peppy's blog](https://blog.ppy.sh) about this, coming soon!\
+\
```

Comment:
isn't this wrong? shouldn't regular paragraphs end with nothing?

Reply:
depends on what was the intention. I see that on the web it produces a larger gap between the paragraphs, so I decided to keep it at that